### PR TITLE
Build date consistently (in term of tz) when executing some embedded JS (bug 1934157)

### PR DESCRIPTION
**Pull request by calixteman**
The date was create in UTC+0 and then amended in using set-Month/Date which take into account the user timezone.
With this patch we build all the date in the user timezone.

### CODE REVIEW

File: src/scripting_api/util.js [hunk 1]
```diff
@@ -409,10 +409,17 @@ class Util extends PDFObject {
     const number = /\d+/g;
     let i = 0;
     let array;
-    const date = new Date(0);
+    const data = {
+      year: 2000,
```
Comment (Snuffleupagus) [inline src/scripting_api/util.js:413]: How was this value chosen, or does it the exact value not matter?

Comment (calixteman) [reply src/scripting_api/util.js]: I just checked in Acrobat and the current year is used, so let's do the same.

**Review by Snuffleupagus — APPROVED** [review]
r=me, assuming that the integration-tests also pass with the latest patch; thank you.

**Pull request merged**